src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -44,7 +44,7 @@ The high-level steps in this tutorial are:
 
 ### Start the primary cluster
 
-To enable physical cluster replication, it is necessary to start each node with the appropriate _configuration profile_ set with the `--config-profile` flag. A configuration profile applies a custom configuration to the server at initialization time. When using physical cluster replication, the `replication-source` and `replication-target` configuration profiles are used to create a virtualized cluster with a system virtual cluster and an application virtual cluster.
+To enable PCR, it is necessary to start each node with the appropriate _configuration profile_ set with the `--config-profile` flag. A configuration profile applies a custom configuration to the server at initialization time. When using PCR, the `replication-source` and `replication-target` configuration profiles are used to create a virtualized cluster with a system virtual cluster and an application virtual cluster.
```

Comment:
@dt didn't we remove this as per https://docs.google.com/document/d/159BfyM_AKay_izAnIpl0BqsKNtWNAJYICYpVFAal3AQ/edit#heading=h.5zgwetg1nme7 ?